assets/queries/dockerfile/missing_dnf_clean_all/query.rego

```diff
@@ -6,7 +6,7 @@ CxPolicy[result] {
 	command := resource.Value[0]
 
 	containsInstallCommand(command)
-
+	not containsDnfClean(input.document[i].command[name], resource.StartLine)
 	not containsCleanAfterInstall(command)
 
 	result := {
```

Comment:
ActualValue should reflect what is happening with the resource. In this case, the clean command is not being run.